packages/format-library/src/link/inline.js

```diff
@@ -35,10 +35,7 @@ const LINK_SETTINGS = [
 	...LinkControl.DEFAULT_LINK_SETTINGS,
 	{
 		id: 'nofollow',
-		title: createInterpolateElement(
-			__( 'Mark as <code>nofollow</code>' ),
-			{ code: <code /> }
-		),
+		title: __( 'Mark as nofollow for search engines' ),
```

Comment:
```suggestion
		title: __( 'Mark as nofollow' ),
```

Sorry - one more tweak to restore original wording 🙏